CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+### 4.4.3
+
+This release of teleport adds enhancements to the Access Workflows API.
+
+* Support for creating limited roles that trigger access requests
+on login, allowing users to be configured s.t. no nodes can be accessed
```

Comment:
What's `s.t.` here? "So that"? "Such that"? We should be explicit and use plain English in release notes.